src/lib/viewers/controls/media/SubtitlesToggle.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import MediaToggle from './MediaToggle';
+import { Subtitle } from './MediaSettingsMenuSubtitles';
+import './SubtitlesToggle.scss';
+
+export type Props = {
+    isShowingSubtitles?: boolean;
+    onSubtitlesToggle?: (isShowingSubtitles: boolean) => void;
+    subtitles?: Array<Subtitle>;
+};
+
+export default function SubtitlesToggle({
+    isShowingSubtitles,
+    onSubtitlesToggle = noop,
+    subtitles = [],
+}: Props): JSX.Element | null {
+    if (subtitles.length <= 0) {
```

Comment:
Looks like we have a few of these checks that could be simplified to `!subtitles.length` or something similar.